app/components/TextEditor/index.js

```diff
@@ -11,23 +11,36 @@ import 'simplemde/dist/simplemde.min.css';
 
 import options from './options';
 
-const TextEditor = props => {
-  const handleEditorChange = txt => {
-    props.onChange(txt);
+/* eslint no-return-assign: "error" */
+class TextEditor extends React.Component {
+  componentDidUpdate() {
+    TextEditor.instance.codemirror.options.readOnly = this.props.disabled;
+  }
+
+  handleEditorChange = txt => {
```

Comment:
I do not thing that this function is needed for SimpleMDE. onChange method of SimpleMDE already does what is needed. {...this.props} should take of mapping component onChange to SimpleMDE onChange.